ArcadiaParties-SPA/src/app/reducers/title-reducers.ts

```diff
@@ -0,0 +1,18 @@
+import { createReducer, on } from '@ngrx/store';
+
+import { changeTitleAction } from '../actions/actions';
+
+export const titleFeatureStateKey = 'title';
+
+export interface TitleState {
+  title: string;
+}
+
+const initialState: TitleState = {
+  title: ''
+};
+
+export const titleReducer = createReducer(
+  initialState,
+  on(changeTitleAction, (state, props) => ({ ...state, title: props.title }))
```

Comment:
You should change state only after success result from title effect. Otherwise it could happpen that title is not changed in the document, but state already contains new title.

Reply:
It fixed